Resuming TARDIS Simulations from HDF/Resuming TARDIS Simulations from HDF.ipynb

```diff
@@ -0,0 +1,1698 @@
+{
```

Comment:
<p>In astronomy, we plot wavelength vs luminosity density because wavelength is the dependent variable. </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/1951/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
Changed plot to better indicate how wavelength is changing relative to luminosity density